dev/source/docs/simulation-2.rst

```diff
@@ -14,12 +14,13 @@ Simulation allows for safe testing of experimental code and settings and crashin
 
 The most commonly used simulators are:
 
--  :ref:`SITL (Software In The Loop) <sitl-simulator-software-in-the-loop>` is the simulator most commonly used by developers.  It is used by the :ref:`autotester <the-ardupilot-autotest-framework>` and other simulators below are actually built on top of SITL
+-  :ref:`SITL (Software In The Loop) <sitl-simulator-software-in-the-loop>` is the simulator most commonly used by developers. It is a simple simulator that is built within all SITL builds of ArduPilot. It is used by the :ref:`autotester <the-ardupilot-autotest-framework>` and other simulators below are actually built on top of SITL
 -  :ref:`Gazebo <using-gazebo-simulator-with-sitl>` is the official DARPA virtual robotics simulator
 -  :ref:`XPlane-10 <sitl-with-xplane>` a commercial flight simulator with a rich 3D interface
 -  :ref:`RealFlight <sitl-with-realflight>` a commercial flight simulator with a rich 3D interface and ability to design custom vehicles
 -  :ref:`Morse <sitl-with-morse>` a robotics simulation environment commonly used in research
--  :ref:`Replay <testing-with-replay>` has no graphical interfacde but allows re-running master from a dataflash log
+-  :ref:`Replay <testing-with-replay>` has no graphical interface but allows re-running master from a dataflash log
```

Comment:
We're trying to avoid "dataflash" now-adays.  More "onboard-log"... preexisting, of course, so no need to fix it here.

Reply:
That was just me fixing up a typo in "interface". Will need a different PR to fix all the dataflash -> onboard-log refs.

Also - is there a compendium somewhere of what "official" words/terms to use in the wiki, like the above example and other things like using "Ardupilot" and not "APM"?

Reply:
Not that I know of.